src/frontend/src/pages/ChangeRequestsPage/ChangeRequestsTable.tsx

```diff
@@ -218,7 +218,7 @@ const ChangeRequestsTable: React.FC = () => {
         sx={{
           border: 0,
           '& .MuiDataGrid-row:hover': {
-            backgroundColor: '#ef4345'
+            backgroundColor: '#7a2122'
```

Comment:
rgba(239, 67, 69, 0.6) can you use this instead of the color you chose. It's the same base color as the hex code that was previously here, but the opacity is at 0.6 instead of 1